sdk/storage/Azure.Storage.Blobs/src/BlobUriBuilder.cs

```diff
@@ -173,9 +185,10 @@ public BlobUriBuilder(Uri uri)
 
                 var startIndex = 0;
 
-                if (uri.IsHostIPEndPointStyle())
+                _isIPStyleUri = uri.IsHostIPEndPointStyle();
+                _isPathStyleUri = _isIPStyleUri || PathStylePorts.Contains(uri.Port);
```

Comment:
nit: can we get rid of `_isIPStyleUri` and do the `uri.IsHostIPEndPointStyle` check inline?

Reply:
Did you want me to fix that nit (and update the changelog as @seanmcc-msft asked), or should I just leave it now that y'all have approved?

Reply:
It would be great if you could fix both real quick.

Reply:
Done!  LMK if there's anything else.  And thank you.

Reply:
@JoshLove-msft, are we good to merge?

Reply:
Yep, LGTM